Ds3/Runtime/Ds3ContentLenghtNotMatch.cs

```diff
@@ -0,0 +1,28 @@
+﻿/*
+ * ******************************************************************************
+ *   Copyright 2014 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+using System;
+using System.Net;
+
+namespace Ds3.Runtime
+{
+    public class Ds3ContentLenghtNotMatch : Exception
+    {
+        public Ds3ContentLenghtNotMatch(string message)
+            : base(message)
```

Comment:
We probably want to add a property that describes how many bytes we read vs how many were expected.